src/System.Security.Cryptography.Xml/tests/Samples/SigningVerifyingX509Cert.cs

```diff
@@ -60,6 +60,13 @@ private static bool VerifyXml(string signedXmlText, X509Certificate2 certificate
         }
 
         [Fact]
+        [SkipOnTargetFramework(
+            TargetFrameworkMonikers.Net45 |
+            TargetFrameworkMonikers.Net451 |
+            TargetFrameworkMonikers.Net452 |
+            TargetFrameworkMonikers.Net46 |
```

Comment:
@bartonjs I wanted to express `<= 4.6.1` regardless of packaging decissions - why do we have those values if we should not be using them anywhere in corefx?